code_samples/simple_language/src/main/java/com/intellij/sdk/language/SimpleRefactoringSupportProvider.java

```diff
@@ -0,0 +1,20 @@
+// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package com.intellij.sdk.language;
+
+import com.intellij.lang.refactoring.RefactoringSupportProvider;
+import com.intellij.psi.PsiElement;
+import com.intellij.sdk.language.psi.SimpleProperty;
+import org.jetbrains.annotations.*;
+
+public class SimpleRefactoringSupportProvider extends RefactoringSupportProvider {
+  @Override
+  public boolean isMemberInplaceRenameAvailable(@NotNull PsiElement elementToRename, @Nullable PsiElement context) {
+    return (elementToRename instanceof SimpleProperty);
+  }
+}
+
+/*
```

Comment:
??

Reply:
One of the errors I occasionally see regarding Kotlin. I have removed it.